source/beamline/CLS/CLSBiStateControl.cpp

```diff
@@ -79,6 +96,29 @@ void CLSBiStateControl::onStateChanged()
 		emit movingChanged(isMoving_);
 }
 
+void CLSBiStateControl::onMoveTimerTimedout()
+{
+	moveTimeoutTimer_.stop();
+
+	if(!moveInProgress_) {
+		// This shouldn't happen
```

Comment:
Although it shouldn't happen, can it happen?  If so, maybe we should put an error mon here?